src/pages/[guild]/leaderboard/[pointsId].tsx

```diff
@@ -57,16 +56,15 @@ const Leaderboard = () => {
     false
   )
 
-  useScrollEffect(() => {
-    if (
-      !wrapperRef.current ||
-      wrapperRef.current.getBoundingClientRect().bottom > window.innerHeight ||
-      data?.leaderboard?.length <= renderedUsersCount
-    )
-      return
-
-    setRenderedUsersCount((prevValue) => prevValue + BATCH_SIZE)
-  }, [data, renderedUsersCount])
+  const disableRendering = useMemo(
```

Comment:
Do we need these useMemos? If not, it'd be much nicer to just inline the condition when passing the prop. If we do, consider the PR approved by me